doc/man/8/cephadm.rst

```diff
@@ -194,6 +194,16 @@ Arguments:
 * [--skip-firewalld]           Do not configure firewalld
 * [--skip-pull]                do not pull the latest image before adopting
 
+Configuration:
+
+When starting the shell, cephadm looks for configuration in the following order:
+
```

Comment:
When starting the shell, cephadm looks for configuration in the following order.  Only the first values found are used.

Reply:
fixed :+1: